tests/foreman/ui/test_rhcloud_insights_vulnerability.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for RH Cloud - Vulnerability
+
+:Requirement: RHCloud
+
+:CaseAutomation: Automated
+
+:CaseComponent: RHCloud
+
+:Team: Proton
+
+:CaseImportance: High
+
+"""
+
+import pytest
+
+
+def create_insights_vulnerability(host):
+    """Function to create vulnerabilities for IoP Vulnerability Engine"""
+    # Upload insights data to Satellite
+    assert host.execute('insights-client --offline').status == 0
```

Comment:
```suggestion
    assert host.execute('insights-client').status == 0
```
`insisghts-client` should run without any specific flags.